lib/ptolemy.ex

```diff
@@ -94,26 +95,29 @@ defmodule Ptolemy do
   ## Example
   ```elixir
   iex(1)> Ptolemy.create(server, :kv_engine1, [:ptolemy, %{test: "foo"}])
-  :ok 
+  :ok
   ```
-  
+
   :gcp_engine
   """
```

Comment:
An example code snippet for GCP here would be nice  💯

Reply:
👍 added docs for GCP